src/xcode/ENA/ENA/Source/Scenes/ErrorReporting/ErrorReportHistory/__tests__/ErrorReportHistoryViewModelTests.swift

```diff
@@ -0,0 +1,44 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+class ErrorReportHistoryViewModelTests: XCTestCase {
+
+	let model = ErrorReportHistoryViewModel()
+	
+	func testSectionsOfViewModel() throws {
+		// GIVEN
+		let tableViewModel = model.dynamicTableViewModel
+		
+		// WHEN
+		let numberOfSections = tableViewModel.content.count
+		
+		// THEN
+		XCTAssertEqual(numberOfSections, 2)
+	}
+	
+	func testForNumberOfStaticCells() throws {
+		// GIVEN
+		let tableViewModel = model.dynamicTableViewModel
+
+		// WHEN
+		let numberOfCells = tableViewModel.section(0).cells.count
```

Comment:
we have a helper method for this inside the dynamicTableViewModel class somthing like 'NumberOfCellsInSection'

Reply:
Done